config/symbols.us.stsel.txt

```diff
@@ -1505,7 +1505,6 @@ func_801B2F50 = 0x801B2F50;
 func_801B3120 = 0x801B3120;
 func_801B3164 = 0x801B3164;
 func_801B367C = 0x801B367C;
-TryLoadSaveData = 0x801B3694;
```

Comment:
Did the formatter remove this? Seems like it's required

```
src/st/sel/2D260.c:(.text+0x3e98): undefined reference to `func_801B3694'
```

Reply:
Yes, that was the formatter, not me. Seems to think it's an orphan symbol.